substrate-node/pallets/pallet-tfgrid/src/interface.rs

```diff
@@ -5,7 +5,7 @@ use frame_support::{
     ensure, sp_runtime::SaturatedConversion, traits::ConstU32, BoundedVec, RuntimeDebug,
 };
 use scale_info::TypeInfo;
-use valip::{ip4::ip::IPv4, ip6::Ip as IPv6, mac::Mac};
+use valip::{ip4::Ip, ip6::Ip as IPv6, mac::Mac};
```

Comment:
I think it would be better to use `ip4::Ip as IPv4`, that way it is consistent with how IPv6 is used, and also with the std library